wordpress-mobile/WordPress-iOS
# Replace occurrences of 'sign in' with 'log in'

Replaces occurrences of "Sign in" with "Log in" so as to be consistent with wp-admin, WordPress.com, and mobile FAQ.

![login signin examples](https://cloud.githubusercontent.com/assets/517257/19745256/23193374-9b8e-11e6-96ec-3cd27cad4396.jpg)

To test:

- Primarily, the login/NUX screens should now say "Log in". 
- The 3D touch shortcut when logged out should say "Log in"
- There are also a few user-facing errors that mention "sign in".

What do you think about this, @mattmiklic ?

Needs review: @aerych
